src/components/Main.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
- [ ] Inside the components folder there should be a a `BOOKS`(_you can name whatever you want_) component which would render all the books using the `BOOK`(_again you can name this as you like_) component
- [ ] The `BOOK` component  takes the book data as a prop and renders the title and the author and has a button for removing books(currently done by you but it should be moved in the components folder side)
- [ ] The AddBook component should also be moved in the component folder side and you should have two inputs one for the author and the other for the title